internal/transport/transport_test.go

```diff
@@ -2052,6 +2058,7 @@ func setUpHTTPStatusTest(t *testing.T, httpStatus int, wh writeHeaders) (stream
 		t.Fatalf("Failed to listen. Err: %v", err)
 	}
 	server = &httpServer{
+		mu:         sync.Mutex{},
```

Comment:
This is unnecessary.  It's automatically initialized to its zero value.

Reply:
Done.